tests/virt_autotest/hotplugging_utils.pm

```diff
@@ -0,0 +1,109 @@
+# SUSE's openQA tests
+#
+# Copyright 2020 SUSE LLC
```

Comment:
As a new module the date can be ommitted. see _CONTRIBUTING.md_

Reply:
I will remove the year.